backend/src/routes/v0/thread.py

```
@@ -357,3 +370,114 @@ async def delete_thread(
+
+    Returns:
+        InterruptResponse with the resolution status
+
+    Raises:
+        404: Interrupt not found
+        403: User doesn't own this thread
+        400: Invalid request (expired, already resolved, validation failed)
+    """
+    try:
+        async with get_checkpoint_db() as checkpointer:
+            service_context = ServiceContext(
```

Ruff (0.14.11)

Local variable `service_context` is assigned to but never used

Remove assignment to unused variable `service_context`

(F841)